changelogs/fragments/6-add-proxmox-cluster-module.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - proxmox_cluster - Add new module proxmox_cluster for creating/joining PVE clusters (https://github.com/ansible-collections/community.proxmox/pull/6).
```

Comment:
Generally new plugins/modules don't need a changelog fragment. It will be automatically created from the top-level `version_added` combined with the `short_description` field.

Reply:
Thanks, just saw that it failed when slurping the fragment in the test pipeline - removed it.